docs/dev/agent_integration.md

```diff
@@ -6,12 +6,24 @@ aliases:
   - /developers/integrations/testing/
   - /integrations/datadog_checks_dev/
   - /guides/new_integration/
-
+  - /developers/integrations/new_check_howto/
+dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/agent_integration.md"
 ---
 
 ## Overview
 
-This guide provides instructions for creating a Datadog Agent integration in the `integrations-extras` repository. For more information about why you would want to create an Agent-based integration, see [Creating your own solution][1].
+This guide provides instructions for creating a Datadog Agent integration. You can list an Agent integration for a price on the Datadog Marketplace, or list it out-of-the-box on the Integrations page. 
+
+## Development Process
+
+You can expect the following process for building an Agent-based integration:
+1. Meet with the Datadog Technology Partner team to discuss your offering and use cases.
+2. Request a sandbox account for development.
+3. Begin development of your integration, which will include writing the integration code on your end as well as building and installing a python wheel (.whl).
+4. Test your integration in your sandbox account.
+5. Once your development work is tested and complete, follow the steps to [Create a Tile][24] in order to display your offering on the Marketplace or Integrations page.
```

Comment:
Not sure if we should include a link to the 'Create a Tile' page since the overview section of that page specifically calls out 'For all offerings that do not use the Datadog Agent,....'